### ISSUE TITLE
Notifications not working with Node v8.4.0

### ISSUE THREAD

**Issue by jackmu95** [JeffreyWay/laravel-mix#1166]
- Laravel Mix Version: v1.4.2
- Node Version: v8.4.0
- NPM Version: v5.4.0
- OS: macOS Sierra (10.12.6)

### Description:

With notifications enabled I get following error when running Laravel Mix on Node v8.4.0:
```bash
internal/child_process.js:325
    throw errnoException(err, 'spawn');
    ^

Error: spawn EACCES
    at _errnoException (util.js:1041:11)
    at ChildProcess.spawn (internal/child_process.js:325:11)
    at exports.spawn (child_process.js:493:9)
    at Object.exports.execFile (child_process.js:208:15)
    at Object.module.exports.fileCommand (node_modules/node-notifier/lib/utils.js:53:13)
    at NotificationCenter.notify (node_modules/node-notifier/notifiers/notificationcenter.js:66:11)
    at module.exports.WebpackNotifierPlugin.compilationDone (node_modules/webpack-notifier/index.js:62:18)
    at Compiler.applyPlugins (node_modules/tapable/lib/Tapable.js:61:14)
    at emitRecords.err (node_modules/webpack/lib/Compiler.js:268:11)
    at Compiler.emitRecords (node_modules/webpack/lib/Compiler.js:375:38)
    at emitAssets.err (node_modules/webpack/lib/Compiler.js:262:10)
    at applyPluginsAsyncSeries1.err (node_modules/webpack/lib/Compiler.js:368:12)
    at next (node_modules/tapable/lib/Tapable.js:218:11)
    at Compiler.compiler.plugin (node_modules/webpack/lib/performance/SizeLimitsPlugin.js:99:4)
    at Compiler.applyPluginsAsyncSeries1 (node_modules/tapable/lib/Tapable.js:222:13)
    at Compiler.afterEmit (node_modules/webpack/lib/Compiler.js:365:9)
    at require.forEach.err (node_modules/webpack/lib/Compiler.js:354:15)
    at node_modules/async/dist/async.js:421:16
    at iteratorCallback (node_modules/async/dist/async.js:998:13)
    at node_modules/async/dist/async.js:906:16
    at node_modules/graceful-fs/graceful-fs.js:43:10
    at FSReqWrap.oncomplete (fs.js:135:15)
npm ERR! code ELIFECYCLE
npm ERR! errno 1
```

A temporary workaround is to set `mix.disableNotifications()` in `webpack.mix.js`.

**Comment by ruchern** [comment]
Permissions error. Can you delete `node_modules` directory and run `npm install` or `yarn` again to see if it works?

**Comment by jackmu95** [comment]
@ruchern I have already done that and it had no affect.

**Comment by ruchern** [comment]
Okay, just letting you know that I tried with:

laravel-mix v1.4.2
npm v5.4.0
node v8.4.0

and notifications are working fine.

**Comment by jackmu95** [comment]
Seems to be related to https://github.com/npm/npm/issues/18399.

**Comment by ruchern** [comment]
Yes, it does seems like an npm issue. Strangely, I am on v5.4.0 and I do not have problems with executing binaries.

**Comment by jackmu95** [comment]
@ruchern What operating system are you using?

**Comment by ruchern** [comment]
@jackmu95 macOS 10.13 Developer Beta 9.

**Comment by jackmu95** [comment]
@ruchern Hmm weird. I'am using npm v5.3.0 for now.

**Comment by ruchern** [comment]
Related and duplicate of #1065

**Comment by jackmu95** [comment]
This error got fixed with npm v5.4.1.

**Comment by ruchern** [comment]
@jackmu95 Yes. I have updated it in #1065
